pyecore/resources/json.py

```diff
@@ -174,7 +176,8 @@ def process_inst(self, inst, features, owning_feature=None):
                 inst.eSet(feature, element)
             elif isinstance(value, list):
                 elements = [self.to_obj(x, owning_feature=feature)
-                            for x in value]
+                            if isinstance(feature, EReference)
```

Comment:
Could you extract the `if` from inside the list comprehension to outside so the condition is not checked for each value (that could cause performance issues)

Reply:
Yhea. I'm not proud of this mistake. My bad :pray: